cmd/kuberhealthy/kuberhealthy.go

```diff
@@ -348,11 +356,9 @@ func (k *Kuberhealthy) monitorExternalChecks(ctx context.Context, notify chan st
 		<-c
 		log.Debugln("Change notification received. Scanning for external check changes...")
 
-		// fetch all khcheck resources from all namespaces
-		l, err := khCheckClient.List(metav1.ListOptions{}, checkCRDResource, "")
+		khChecks, err := listUnstructuredKHChecks()
 		if err != nil {
-			log.Errorln("Error listing check configuration resources", err)
-			continue
+			log.Errorln("error listing unstructured khChecks: %w", err)
```

Comment:
would it make sense to `continue` here?  Do we want to keep doing things in this iteration if we were unable to list unstructured checks?

Reply:
yup, you're right, there should be a `continue` here